test/ci_app_tests/test_tau_service.py

```diff
@@ -0,0 +1,31 @@
+# report / C config test
+
+import unittest
+
+import calipertest as cat
+
+class CaliperReportTest(unittest.TestCase):
+    """ Caliper TAU service test case """
+
+    def test_tau(self):
+        target_cmd = [ './ci_test_tau_service' ]
+        query_cmd  = [ '../../src/tools/cali-query/cali-query',
+                       '-q', 'SELECT count(),iteration#fooloop WHERE loop=fooloop GROUP BY iteration#fooloop FORMAT expand' ]
+
+        caliper_config = {
+            'CALI_SERVICES_ENABLE'   : 'event,trace,report,tau',
+            'CALI_REPORT_CONFIG'     : 'format cali',
+            'CALI_LOG_VERBOSITY'     : '0'
+        }
+
+        #query_output = cat.run_test_with_query(target_cmd, query_cmd, caliper_config)
+        query_output = cat.run_test(target_cmd, caliper_config)
+        #snapshots = cat.get_snapshots_from_text(query_output)
```

Comment:
Not quite sure what this does ...

The idea with the python unit test framework is that we can run tests with a given config and check the Caliper output. This one doesn't really check the tau service output. It would detect if it crashes or something, so that may still be good to have. In that event you don't need the event,trace, and report services though.